infrastructure/terraform/components/reporting/autoscaling_schedule_scale_in.tf

```diff
@@ -1,10 +1,11 @@
 resource "aws_autoscaling_schedule" "scale_in" {
-  count = var.enable_powerbi_gateway ? 1 : 0
+  count = var.enable_powerbi_gateway && var.scale_in_recurrence_schedule != null ? 1 : 0
```

Comment:
what happens when var.enable_powerbi_gateway is true but var.scale_in_recurrence_schedule isnt set with these changes